tests/pubgrade/test_server.py

```diff
@@ -252,141 +278,268 @@ def mock_notify_subscriptions():
+        "users"
+    ].client = mongomock.MongoClient().db.collection
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "users"
+    ].client.insert_one(MOCK_USER_DB)
+
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "admin_users"
+    ].client = mongomock.MongoClient().db.collection
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "admin_users"
+    ].client.insert_one(MOCK_ADMIN_USER_1)
+
+    with app.test_request_context(
+        json={},
+        headers={
+            "X-Super-User-Id": MOCK_ADMIN_USER_1["uid"],
+            "X-Super-User-Access-Token": MOCK_ADMIN_USER_1[
+                "user_access_token"
+            ],
+            "Content-Type": "application/json",
+        },
+    ):
+        res = getUsers.__wrapped__()
+        assert res[0]["name"] == "Akash"
+        assert isinstance(res, list)
+
+
+def test_verify_user():
+    app = Flask(__name__)
+    app.config["FOCA"] = Config(
+        db=MongoConfig(**MONGO_CONFIG), endpoints=ENDPOINT_CONFIG
+    )
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "users"
+    ].client = mongomock.MongoClient().db.collection
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "users"
+    ].client.insert_one(MOCK_USER_DB)
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "admin_users"
+    ].client = mongomock.MongoClient().db.collection
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "admin_users"
+    ].client.insert_one(MOCK_ADMIN_USER_1)
+
+    with app.test_request_context(
+        json={},
+        headers={
+            "X-Super-User-Id": MOCK_ADMIN_USER_1["uid"],
+            "X-Super-User-Access-Token": MOCK_ADMIN_USER_1[
+                "user_access_token"
+            ],
+            "Content-Type": "application/json",
+        },
+    ):
+        verifyUser.__wrapped__(MOCK_USER_DB["uid"])
+
+
+def test_unverify_user():
+    app = Flask(__name__)
+    app.config["FOCA"] = Config(
+        db=MongoConfig(**MONGO_CONFIG), endpoints=ENDPOINT_CONFIG
+    )
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "users"
+    ].client = mongomock.MongoClient().db.collection
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "users"
+    ].client.insert_one(MOCK_USER_DB)
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "admin_users"
+    ].client = mongomock.MongoClient().db.collection
+    app.config["FOCA"].db.dbs["pubgradeStore"].collections[
+        "admin_users"
+    ].client.insert_one(MOCK_ADMIN_USER_1)
+
+    with app.test_request_context(
+        json={},
+        headers={
+            "X-Super-User-Id": MOCK_ADMIN_USER_1["uid"],
+            "X-Super-User-Access-Token": MOCK_ADMIN_USER_1[
+                "user_access_token"
+            ],
+            "Content-Type": "application/json",
+        },
+    ):
+        unverifyUser.__wrapped__(MOCK_USER_DB["uid"])
```

Comment:
Also maybe this needs to have an assert or similar to the check the return value?

Reply:
ok solved.